[Marketplace Contribution] Splunk - Content Pack Update (#41317)
demisto/content: Packs/SplunkPy/ReleaseNotes/3_2_21.md

#### Integrations

##### SplunkPy

> - Added support for **splunk-job-share** command that change job settings to share its results to all splunk users, and change its ttl.

Reviewer: ```suggestion
- Added a new command ***splunk-job-share*** that change job settings to share its results to all Splunk users, and changed its TTL.
```
